v1/serviceEntity/config-file.yaml

```diff
@@ -0,0 +1,21 @@
+title: ConfigFile
+type: object
+required:
+- id
```

Comment:
Do we need to make the id required? If its not needed then lets make it non required.

Reply:
Yes @brijeshdhakar in override we override config file based on id.